scripts/modules/MsIndustryLinks/azureDeploymentPackage/azureblobstorage/connection.json

```diff
@@ -0,0 +1,34 @@
+{
+  "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentTemplate.json#",
```

Comment:
Is there a reason why we have gone with ARM templates over bicep?

Reply:
Bicep files are hard to customize programmatically. In the end, I'd have to compile into ARM template to customize anyways.